synthesizer/sed.py

```diff
@@ -43,17 +36,31 @@ class Sed:
     fnu = Quantity()  # nJy
```

Comment:
Some quantities are missing: z variants and broadband quantities

Reply:
I've updated this and changed `nuz` -> `obsnu`. 

Which broadband quantities do you mean? I think things like bolometric_luminosity don't need to be specified as a quantity since we never use the unit-less versions, i.e. they are always returned with units of e.g. `luminosity.units`.

Reply:
I had been poking at the broadband quantities. It would mean modifying a `Quantity` to understand when things are dictionaries. Maybe worthwhile in the long run but I won't bother for now.

Reply:
OK.